ninja-core/src/site/markdown/team.md

```diff
@@ -10,10 +10,17 @@ new releases. And driving the platform as a whole.
 That leadership role is currently performed by:
 
  * The team of [FinalFrontierLabs](http://www.finalfrontierlabs.com) 
-   and [Raphael](http://raphaelbauer.com) as lead architect. 
+   and [Raphael](http://raphaelbauer.com) as lead architect.
+
  * The team of [Fizzed, Inc.](http://fizzed.com) and [Joe Lauer](http://lauer.bz)
    as lead architect.
 
+Ninja is proudly sponsored by <a href="https://www.greenback.com">Greenback</a>
```

Comment:
That's 100% cool - it's great you and your team is part of the project! All the very very best for greenback.com!